ansible/playbook/tasks/update.yml

```diff
@@ -124,6 +124,12 @@
     - name: Create working directory for Alertmanager
       file: path=/srv/alertmanager/data state=directory owner=pmm group=pmm
 
+    - name: Create working directory for VictoriaMetrics
+      file: path=/srv/victoriametrics/data state=directory owner=pmm group=pmm
+
+    - name: Create empty configuration file for VictoriaMetrics
+      file: path=/etc/victoriametrics-promscrape.yml state=touch owner=pmm group=pmm
```

Comment:
Could it be added checks that folder and file have not existed yet?

Reply:
I suppose, that file module already checks it and create directory only if its not exists. state=touch does the same.